script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2B.txt

```diff
@@ -1,15 +1,15 @@
 <Page1>
 After having breakfast in the dining room, I go out to the lobby.
-Behind me is Hisui, carrying my bag.
+Behind me is Hisui, carrying my bag.//　後ろには鞄を持った翡翠が付き添ってくれている。 "添ってくれている。" intentionally left out.
 I suggested that I could carry my own bag, but she insisted on carrying it to the door at least.
 <Page2>
 ...I seem to have a weak spot not only for Akiha, but Hisui as well.
-Oh, well, a soft spot would probably be more precise.
+Oh, well, a soft spot would probably be more accurate.//　あ、いや、正しくは、翡翠には甘い、かもしれない。liberally translated
 <Page3>
 In Akiha's case, she is simply correct so I lose in terms of reasoning. It is also true that Tohno Akiha's position is above Tohno Shiki's.
 <Page4>
-But Hisui is the complete opposite.
-I'm in a position of power over Hisui, but I don't think I can use that power to my advantage.
+But Hisui is nothing like that.//　けれど翡翠は真逆だ。
```

Comment:
I kinda like opposite here, since there's a direct inverse correlation betwen Akiha > Skihi and Shiki > Hisui in the pecking order